code/style.css

```diff
@@ -1,5 +1,93 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
-
-body {
-  background-color: lightblue;
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+ul{
+    display: flex;
+    list-style: none;
+    margin: 10px;
+    float: right;
+}
+li{
+    font-size: 14px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 400;
+    color: rgb(0, 0, 0,0.85);
+    margin: 15px;
+    cursor: pointer;
+}
+li:hover{
+    text-decoration: underline;
+}
+#signIn_button{  
+    background-color: rgb(72, 116, 236);
+    color: #fff;
+    border: 10px solid  rgb(72, 116, 236);
+    border-radius: 3px;
+    text-decoration: none;
+}
+#signIn_button:hover{
+    background-color:  rgba(77, 127, 253, 0.8);
+    border: 10px solid  rgba(77, 127, 253, 0.849);
+    text-decoration: none;
+}
```

Comment:
Nice that you use hover and change the background color! 
You may also add `cursor: pointer;`